source/faq.txt

```diff
@@ -136,6 +136,14 @@ These logs do not indicate that ``mongosync`` is failing or corrupting
 data. If a fatal error occurs, ``mongosync`` stops the sync and writes a
 fatal log entry. 
 
+What should I do if ``mongosync`` returns a fatal error?
+--------------------------------------------------------
+
+A fatal error indicates a problem that must be addressed and requires the 
```

Comment:
Rewrite to not use address, which has multiple meanings and is not global. Example, use "fixed" or "examined" or similar.

Reply:
Updated.